packages/react/src/DataTable/DataTable.features.stories.tsx

```diff
@@ -123,7 +123,7 @@ const data: Array<Repo> = [
     id: 8,
     name: 'docker-templates',
     type: 'public',
-    updatedAt: now - 3 * Month,
```

Comment:
the difference in how months/days are rendered will cause the table to grow/shrink depending on the date, which is going to constantly generate VRT failures. setting to static date instead